src/main/java/teammates/common/datatransfer/questions/FeedbackMcqQuestionDetails.java

```diff
@@ -565,6 +567,20 @@ public String getQuestionTypeChoiceOption() {
                         + Const.FeedbackQuestion.MCQ_MIN_NUM_OF_CHOICES + ".");
             }
 
+            // If there are Empty Mcq options entered, at least two of the Mcq options must be filled, given that the
```

Comment:
Is empty MCQ option allowed in the first place?

Reply:
The user can leave the mcq option blank or the case when the user just enters blank spaces in the box

Reply:
Then safe to say you want to just prohibit empty options altogether? Otherwise this can be easily gamed with e.g. 2 valid options and 100 empty strings.

Reply:
> 2 valid options and 100 empty strings.
Exactly. I'll update the validation to prohibit all empty options